doc/install/index.rst

```diff
@@ -59,6 +59,8 @@ tool that can be used to quickly deploy clusters. It is deprecated.
 
 `github.com/openstack/puppet-ceph <https://github.com/openstack/puppet-ceph>`_  installs Ceph via Puppet.
 
+`opennebula HCI clusters <https://docs.opennebula.io/stable/provision_clusters/hci_clusters/overview.html>`_ installs Ceph on different cloud providers
```

Comment:
A dot is missing at the end of the sentence.  Also, should opennebula be capitalized here?
```suggestion
`OpenNebula HCI Clusters <https://docs.opennebula.io/stable/provision_clusters/hci_clusters/overview.html>`_ installs Ceph on different cloud providers.
```

Reply:
... install Ceph on various cloud platforms.

Reply:
Accepted.